Source/SteveSloth/Private/MyEnemy.cpp

```diff
@@ -0,0 +1,39 @@
+/****************************************************************************************
+
+Copyright: SteveSloth
+
+Name: Aidan
+
+Script: MyEnemy.cpp
+
+Date: April 23. 2024
+
+Description: This is the base class for all Enemies.
+
+TODO:
+
+Known Bugs:
+
+TODO:
+
+Known Bugs:
+****************************************************************************************/
+
```

Comment:
Please check the Whitespacing above and fix. Code should always be kept neat, clean and organized with a good flow